src/estimators/tests/exact_set_test.py

```diff
@@ -22,43 +22,66 @@
 import numpy as np
 
 from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import AddRandomElementsNoiser
-from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import ExactSet
+from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import ExactMultiSet
 from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import LessOneEstimator
 from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import LosslessEstimator
 
 
-class ExactSetTest(absltest.TestCase):
+class ExactMultiSetTest(absltest.TestCase):
```

Comment:
It would be nice to add a unit test for testing LosslessEstimator and LessOneEstimator estimating an empty exact multiset (an exact multiset which no IDs are added to it).